### ISSUE TITLE
[Feature request] Ability to lock a specific cat in

### ISSUE THREAD

**Issue by jacotec** [PetHubLocal/pethublocal#4]
Hi Peter,
I hope it's OK to also collect ideas and feature requests here? If not, please delete it ;-)

The Sure Petcare cloud gives the ability to change the outdoor permission per cat. This is a great feature i.e. if one of our furry family members has an appointment at the vet. Unfortunately that switch is missing in PetHubLocal. Might be something for the list :-)

**Comment by PapaErde** [comment]
Hi Jacotec,
without a dual-scan flap, this feature is not possible. why? the software hast to know which one of the cats stands in front of the flap (inside). but the scanning coil is in the middle of the flap.  but if you own the sureflap dualscan door, it´s only a question of the right code. just my 2c´s. .greez

**Comment by jacotec** [comment]
@PapaErde Yes, but the CatFlap Connect, which works with the Hub, is a dual scan type.
The pet door can't, but the cat flap can ... so it should be supported IMHO. That feature is pretty essential.

**Comment by plambrechtsen** [comment]
I have a stack of changes I need to commit and after I have changed the animal state to a switch I will look to add this feature

**Comment by plambrechtsen** [comment]
Closing this issue as I now have a whole new approach. This python code is great for a reference but I won't be making any changes to it going forward. If you are interested in my new approach send me an email pethublocal at gmail . com.
